.github/workflows/llm_integration.yml

```diff
@@ -584,6 +584,33 @@ jobs:
           serve
           python3 llm/client.py lmi_dist_adapters llama-7b-unmerged-lora-overflow
           docker rm -f $(docker ps -aq)
+      - name: Test lmi-dist lora awq - llama2-13b
```

Comment:
given we have a growth of LoRA tests. I would suggest we create another job to handle all LoRA tests